.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -318,6 +323,14 @@ jobs:
           name: "${{ steps.build.outputs.slsa-outputs-name }}-${{ needs.rng.outputs.value }}"
           path: "${{ steps.build.outputs.slsa-outputs-name }}"
 
+      - name: Upload output artifacts
```

Comment:
no need for a secure upload since the predicate file is securely uploaded, hence verification would fail. Correct?

Reply:
yeah, the idea here was that the slsa layout was uploaded. but thinking about it more, I think I will secure upload it. I'll need to download them and upload it to a release in a later job.

Reply:
I'm filing a follow-up, secure-upload doesn't support folders to compute sha256